constants_local.xml

```diff
@@ -37,6 +37,7 @@
         <TXN_SUBMISSION>5</TXN_SUBMISSION>
         <TXN_BROADCAST>10</TXN_BROADCAST>
         <DEBUG_LEVEL>3</DEBUG_LEVEL>
+        <FULL_DATASET_MINE>0</FULL_DATASET_MINE>
```

Comment:
This option should belongs to "switch", so I suggest to move to part.

Reply:
In the future, there should be only full dataset mining, it is not an option for user.